app/src/main/java/app/revanced/manager/ui/viewmodel/ContributorViewModel.kt

```diff
@@ -11,13 +14,13 @@ import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 
 class ContributorViewModel(private val reVancedAPI: ReVancedAPI) : ViewModel() {
-    val repositories = mutableStateListOf<ReVancedGitRepository>()
+    var repositories: List<ReVancedGitRepository>? by mutableStateOf(null)
 
     init {
         viewModelScope.launch {
-            withContext(Dispatchers.IO) { reVancedAPI.getContributors().getOrNull() }?.let(
-                repositories::addAll
-            )
+            withContext(Dispatchers.IO) {
+                repositories = reVancedAPI.getContributors().getOrNull()
+            }
```

Comment:
This should be done on the main thread.
```suggestion
            repositories = withContext(Dispatchers.IO) {
                reVancedAPI.getContributors().getOrNull()
            }
```